### PR TITLE
refac: Folder Structure

### PR DESCRIPTION
**Pull request by ShaBloze**

### CODE REVIEW

**Review by BjarkeMeier — APPROVED** [review]

**Review by Gizmooa — COMMENTED** [review]
The `readme` needs to be updated to reflect these changes. When that is fixed, LGTM 😄

File: README.md [hunk 1]
```diff
@@ -54,3 +59,7 @@ Documentation folder should contain all necessary documentation to onboard devel
 ### source
 
 Source folder should contain all sourcecode in this repository.
```
Comment (Gizmooa) [inline README.md:61]:
```suggestion
```

Isn't it pretty much the same under here? 😄

Comment (Gizmooa) [inline README.md:61]:
```suggestion
```

Isn't it pretty much the same as beneath?

**Review by Gizmooa — APPROVED** [review]
LGTM!

**Review by NicolajAaH — APPROVED** [review]

**Pull request merged**